src/components/AnchorLink.js

```diff
@@ -27,7 +28,7 @@ export function AnchorLink({
   if (title) linkProps.title = title;
   if (className) linkProps.className = className;
 
-  return <Link {...linkProps}>{Boolean(children) ? children : title}</Link>;
+  return <Link {...linkProps, ...additionalLinkProps}>{Boolean(children) ? children : title}</Link>;
```

Comment:
`onClick` would be overridden and this component's behavior wouldn't work properly. What if `onClick` is destructured from props and is tail-called in this component's `onClick`'s handler?